src/blocks/slider/Slider.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+
+import { Slider as MuiSlider } from '@material-ui/core';
```

Comment:
Sir we will be removing Material UI from our codebase in future, so this isn't gonna work. I guess we discussed this earlier.

We can only use those 3rd part libraries which offers standalone components like react and radix and not the whole design system like material UI.